build-cake.ps1

```diff
@@ -0,0 +1,5 @@
+# Hack until dotnet/cli##1701 is accepted
```

Comment:
Oh I see you preserved the old build files, I'm fine with getting rid of them to replace with Cake assuming all the features are replaced, we should have just one set of build files.